processed 12 tasks

init:
A: object(0,0)

task 1 'publish'. lines 8-79:
created: object(1,0)
mutated: object(0,1)
gas summary: computation_cost: 1000000, storage_cost: 9963600,  storage_rebate: 0, non_refundable_storage_fee: 0

task 2 'run'. lines 80-82:
Error: Transaction Effects Status: Move object with size 256001 is larger than the maximum object size 256000
Execution Error: ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: MoveObjectTooBig { object_size: 256001, max_object_size: 256000 }, source: None, command: None } }

task 3 'run'. lines 83-85:
created: object(3,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000000, storage_cost: 25625,  storage_rebate: 13, non_refundable_storage_fee: 0

task 4 'run'. lines 86-88:
created: object(4,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000000, storage_cost: 25625,  storage_rebate: 13, non_refundable_storage_fee: 0

task 5 'run'. lines 89-91:
Error: Transaction Effects Status: Move object with size 256001 is larger than the maximum object size 256000
Execution Error: ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: MoveObjectTooBig { object_size: 256001, max_object_size: 256000 }, source: None, command: None } }

task 6 'run'. lines 92-92:
created: object(6,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000000, storage_cost: 25622,  storage_rebate: 13, non_refundable_storage_fee: 0

task 7 'run'. lines 94-94:
created: object(7,0)
mutated: object(0,0)
wrapped: object(6,0)
gas summary: computation_cost: 50000000, storage_cost: 25626,  storage_rebate: 25622, non_refundable_storage_fee: 0

task 8 'run'. lines 96-101:
Error: Transaction Effects Status: Move object with size 256001 is larger than the maximum object size 256000
Execution Error: ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: MoveObjectTooBig { object_size: 256001, max_object_size: 256000 }, source: None, command: None } }

task 9 'run'. lines 102-104:
Error: Transaction Effects Status: Move object with size 256001 is larger than the maximum object size 256000
Execution Error: ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: MoveObjectTooBig { object_size: 256001, max_object_size: 256000 }, source: None, command: None } }

task 10 'run'. lines 105-107:
created: object(10,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000000, storage_cost: 25625,  storage_rebate: 13, non_refundable_storage_fee: 0

task 11 'run'. lines 108-108:
created: object(11,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000000, storage_cost: 25625,  storage_rebate: 13, non_refundable_storage_fee: 0
